locksmith/openapi.yml

```
@@ -46,3 +46,39 @@ paths:
       responses:
         '200':
           description: OK
+  /v2/member/{network}/locks/{lockAddress}/members:
```

```suggestion
  /v2/member/{network}/locks/{lockAddress}/keys:
```

Ok so now I realize this is not listing members, but keys (tokens). Sorry I was confused earlier because you used `list`.